### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DawerG**

### CODE REVIEW

**Review by 1duo — APPROVED** [review]

**Review by bhushan23 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,34 +2,38 @@
 
 [![Build Status](https://travis-ci.org/onnx/onnx-coreml.svg?branch=master)](https://travis-ci.org/onnx/onnx-coreml)
 
-This tool converts [ONNX](https://onnx.ai/) models to Apple Core ML format. To convert Core ML models to ONNX, use [ONNXMLTools](https://github.com/onnx/onnxmltools).
+## PyTorch Models
```
Comment (srikris) [inline README.md:5]: We should put a disclaimer right on top that this package is no longer supported.

File: README.md [hunk 2]
```diff
@@ -2,34 +2,38 @@
...
 
-Please see the ONNX conversion section in the [Neural network guide](https://github.com/apple/coremltools/blob/master/examples/NeuralNetworkGuide.md) 
-on how to use the converter. 
+`onnx-coreml` package will **no longer be maintained**.
 
-There are a few [notebook examples](https://github.com/apple/coremltools/tree/master/examples/neural_network_inference) 
-as well for reference.  
+Note: ONNX converter is not under any active feature development.
+
+## ONNX Conversion API
```
Comment (srikris) [inline README.md:29]: We should remove the examples and code documentation here. It's confusing. If the package is not supported then it's not clear we should encourage users to use this.

File: README.md [hunk 3]
```diff
@@ -2,34 +2,38 @@
...
 
 
 ### Parameters
 
 ```python
+from coremltools.converters.onnx import convert
```
Comment (srikris) [inline README.md:35]: All of this code should live in the documentation of coremltools.readme.io

File: README.md [hunk 4]
```diff
@@ -121,64 +125,15 @@ __minimum_ios_deployment_target__: str
...
-pytest
-```
-
-To run a specific unit test, for instance the custom layer test, run
-
-```shell
-pytest -s custom_layers_test.py::CustomLayerTest::test_unsupported_ops_provide_functions
+pip install coremltools==4.0b1
 ```
+For the latest changes please see the [release notes](https://github.com/apple/coremltools/releases/).
```
Comment (srikris) [inline README.md:136]: Again, please keep the instructions to document coremltools in that documentation. This will get stale quickly.

**Review by srikris — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -2,34 +2,38 @@
...
 
-This tool converts [ONNX](https://onnx.ai/) models to Apple Core ML format. To convert Core ML models to ONNX, use [ONNXMLTools](https://github.com/onnx/onnxmltools).
+## PyTorch Models
 
-There's a comprehensive [Tutorial](https://github.com/onnx/tutorials/tree/master/examples/CoreML/ONNXLive/README.md) showing 
-how to convert PyTorch style transfer models through ONNX to Core ML models and run them in an iOS app. 
+For converting PyTorch models to CoreML format, the recommended approach is to use **new** PyTorch to Core ML converter, introduced in the [`coremltools 4.0`](https://github.com/apple/coremltools) python package.
 
-To get the latest version of `onnx-coreml` from PyPI:
+Please see the  coremltools documentation on [PyTorch conversion](https://coremltools.readme.io/docs/pytorch-conversion) for example usage.
```
[the remark this answers is not in the document]
Comment (DawerG) [reply README.md]: Done.

File: README.md [hunk 6]
```diff
@@ -2,34 +2,38 @@
...
+
+To get the latest version of coremltools:
 
 ```shell
-pip install --upgrade onnx-coreml
-pip install --upgrade coremltools # onnx-coreml depends on the coremltools package
+pip install coremltools==4.0b1
```
[the remark this answers is not in the document]
Comment (DawerG) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (DawerG) [reply README.md:5]: Done.

File: README.md (hunk 2 above)
Comment (DawerG) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (DawerG) [reply README.md]: Moved.

File: README.md (hunk 4 above)
Comment (DawerG) [reply README.md]: Removed.

**Review by srikris — APPROVED** [review]

**Pull request merged**